doc/data/messages/u/unhashable-dict-key/bad.py

```diff
@@ -0,0 +1,2 @@
+fruits_number = {"apple": 42}
+print(f"Number of apples : {fruits_number[['apple']]}")  # [unhashable-dict-key]
```

Comment:
```suggestion
print(f"Number of apples : {{'apple': 42}[['apple']]}")  # [unhashable-dict-key]
```
The message might only be raised when the dict is defined in place like this ?

Reply:
Looks like it yeah. I've updated the examples but no worries if you would like to change them again - I am not sure the more I think about them :D